lms/lmsweb/config.py.example

```diff
@@ -67,3 +67,5 @@ LIMITS_PER_HOUR = 50
 
 # Change password settings
 MAX_INVALID_PASSWORD_TRIES = 5
+
+REPOSITORY_FOLDER = os.getenv("REPOSITORY_FOLDER", os.path.abspath(os.path.join(os.curdir, "repositories")))
```

Comment:
Non-blocking: Our code doesn't have >79 chars lines, so if you can split it to 2 pathes it'll be great.

```
DEFAULT_REPOSITORY_FOLDER = (Path() / "repositories").resolve()
REPOSITORY_FOLDER = os.getenv("REPOSITORY_FOLDER", DEFAULT_REPOSITORY_FOLDER)
```